deployment/ansible/group_vars/packer

```diff
@@ -17,3 +17,5 @@ tiler_host: "tile.service.nyc-trees.internal"
 postgresql_username: nyctrees
 postgresql_password: nyctrees
 postgresql_database: nyc_trees
+
+tasseo_graphite_url: "http://$(curl -s http://instance-data.ec2.internal/latest/meta-data/public-hostname):8080"
```

Comment:
Hm. This should be `graphite_web_port`, which ends up being whatever `apache_port` is (Graphite's web interface is a Django application that runs with `modwsgi`).

In development, we're bypassing the Vagrant port forwarding because we're using the machine's private host-only interface IP. In testing, monitoring stuff like this is disabled. In production, we just use the default ports for everything because the default ports are hardcoded in the CloudFormation.

Fixed in https://github.com/azavea/nyc-trees/commit/bd14bcd316f7fb62117c3a264377b65c83bebfdf.